driver/makedepend.ml

```diff
@@ -69,7 +69,7 @@ let readdir dir =
       try
         Sys.readdir dir
       with Sys_error msg ->
-        Format.fprintf Format.err_formatter "@[Bad -I option: %s@]@." msg;
+        Format.fprintf ppf "@[Bad -I option: %s@]@." msg;
```

Comment:
I would propose to rename `ppf` to `stderr` or `err` to make clearer that this is a global constant.
Otherwise I was wondering when reading the diff if there was a `ppf` argument in scope.

Reply:
I think I'll just use `Format.eprintf`.